container-image-tags/README.md

```diff
@@ -0,0 +1,30 @@
+# Container Image Tags Action
+
+An action for creating container image tags from branch, tag and pull request
+names.
+
+## Inputs
+
+### `strip-tag-prefix`
+The tag prefix to strip i.e v1.2.3 -> 1.2.3 (default 'v') if GitHub reference is
+a tag.
+
+### `image-name`
+The image name to use. By default it is derived from the repository name.
```

Comment:
```suggestion
The image name to use. By default it is derived from the repository name.
```

e.g.?
`<orga>/<repo>:<branch/tag>`?

Reply:
I've extended the README to explain how the tags are generated.